tools/AppsPackages/src/Views/AppsPackagesView.xaml.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using DevHome.AppsPackages.ViewModels;
+using DevHome.Common;
+
+namespace DevHome.AppsPackages.Views;
+
+public partial class AppsPackagesView : ToolPage
+{
+    public override string ShortName => "AppsPackagesView";
+
+    public AppsPackagesViewModel ViewModel
+    {
+        get;
+    }
+
+    public AppsPackagesView()
+    {
+        ViewModel = new AppsPackagesViewModel();
```

Comment:
Consider resolving the view model from the dependency injection container. Steps below:

1. Resolve type from container (registration in next steps)
```suggestion
        ViewModel = Application.Current.GetService<AppsPackagesViewModel>();
```

2. Add extension class by creating `/Extensions/ServiceExtensions.cs` in the cs project. Add the following method:
```cs
private static IServiceCollection AddAppAndPackages(this IServiceCollection services)
{
    // View models
    services.AddTransient<AppsPackagesViewModel>();
}
```

3. After adding reference from DevHome.csproj to your csproj, add a call to the new extension method:
```cs
services.AddAppAndPackages();
```